templates/user/profile.html

```diff
@@ -0,0 +1,95 @@
+{% extends "base.html" %}
+
+{% block title %}My user profile{% endblock title %}
+
+{% block content %}
+<!-- Begin Page Content -->
+<div class="container-sm py-5">
+
+    <!-- Main Content Here -->
+    <div class="card border-0 shadow p-4">
+        <div class="card-body">
+            <div class="row">
+                <div class="col">
+                    <h3 class="card-title mb-0">My user profile</h3>
+                </div>
+
+                <div class="row">
+                    <div class="col">
+                        <div class="row pt-4">
+                            <h6 class="fw-bold">Account</h6>
+                        </div>
+                        <div class="row pt-1">
+                            <div class="col-4">Email address</div>
+                            <div class="col-8 fw-bold">{{user.email}}</div>
+                        </div>
+
+                        <div class="row pt-4">
+                            <h6 class="fw-bold">Contact</h6>
+                        </div>
+                        <div class="row">
+                            <div class="col-4">First name</div>
+                            <div class="col-8 fw-bold">{{user.first_name}}</div>
+                        </div>
+                        <div class="row pt-1">
+                            <div class="col-4">Last name</div>
+                            <div class="col-8 fw-bold">{{user.last_name}}</div>
+                        </div>
+
+                        <div class="row pt-1">
+                            <div class="col-4">Affiliation</div>
+                            <div id="userAffiliation" class="col-8 fw-bold">{{user_profile.affiliation}}</div>
+                        </div>
+                        <div class="row pt-1">
+                            <div class="col-4">Department</div>
+                            <div class="col-8 fw-bold">{{user_profile.department}}</div>
+                        </div>
+
+                        <div class="row mt-5 p-3 small text-white" style="background-color: #82AEB2;">
```

Comment:
Let's use the css classes instead.
```suggestion
                        <div class="row mt-5 p-3 small text-white bg-teal-050">
```

Reply:
Good suggestions. I will fix this.